acme_diags/derivations/acme.py

```diff
@@ -97,6 +97,14 @@ def qflxconvert_units(var):
         var.units = 'mm/day'
     return var
 
+
+def qflx_convert_to_lhflx(qflx):
```

Comment:
@chengzhuzhang I use this function to help derive `NET_FLUX_SRF` on Line 533 and `LHFLX` on Line 622. If the changes in this file (acme.py) look good to you, then this pull request can be merged.